### PR TITLE
View cell for Survey optionS

### LINKED ISSUE #907: View cell for Survey OptionS column
## Description
The survey options column Works by letting the user pick a survey question (for which the response type is “options”) and renders all options that a person selected for that question.

## Relevant Job Stories
(no specific ones)

## Prerequisites
- Maybe finish #912 first?

## Requirements
- [x] If no response - empty cell
- [x] Number of selected options in right side of ceell
- [x] As many options as fits displayed as chips in cell
- [x] When focused shows "popper" 

## Possible implementations
Use component created in #912 

## Design specifications
https://www.figma.com/file/EjLcpjKWgNizCb5PRBF51P/Zetkin-3?node-id=1976%3A36120&t=ZwOBKqXDfw3cdL7Y-4
bild

### PR DESCRIPTION
**Pull request by rebecarubio**
## Description
This PR implements `survey_options` view cells. It means that it renders all options that a person selected for that question in the view and a counter with the total number of the selected options. Special thanks to @Bagera for make the styles of the chips (they break when they don´t fit in the cell and there´s also a ellipsis when can´t be displayed more options, see screenshots)

## Screenshots
b-1
b-with ellipsis
b-popover

## Changes
* Adds `survey_options` type cell for `survey_options` column and styles for the cell
* Fixes bug in `surveyResponseColumnType` to filter empty cells

## Related issues
Resolves #907

### CODE REVIEW

File: src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            },
+            width: 400,
+          });
+        }}
+        submissions={cell.map((sub, index) => ({
+          id: sub.submission_id,
+          matchingContent:
+            index == 0
+              ? sub.selected.map((s) => (
+                  <Chip key={s.id} label={s.text} variant="outlined" />
```
Comment (richardolsson) [inline src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx:131]: I think these chips need a little margins between them. 
image

**Review by richardolsson — CHANGES_REQUESTED** [review]
I think this looks super good except for a little bit of missing margins between the chips in the preview. Is that something that could be amended?

File: src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx (hunk 1 above)
Comment (rebecarubio) [reply src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx:131]: sure! good idea!

Comment (rebecarubio) [reply src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx]: margins

Comment (richardolsson) [reply src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx]: Nice! 💯

**Review by richardolsson — APPROVED** [review]
It's looking good and I like it! 💯

**Pull request merged**